netci.groovy

```diff
@@ -34,6 +34,10 @@ platformList.each { platform ->
     else if (os == 'Ubuntu') {
         buildCommand = "./build.sh --skip-prereqs --configuration ${configuration} --docker ubuntu.14.04 --targets Default"
     }
+	else if (os == 'Linux') {
+		os = 'Ubuntu16.04';
```

Comment:
@Petermarcu and I agreed to use Ubuntu 16.04 because it is a popular distro for CLI and because it now doubles our distro coverage across corefx and cli.